src/main/java/com/theironyardf/entities/Review.java

```diff
@@ -0,0 +1,80 @@
+package com.theironyardf.entities;
+
+import javax.persistence.*;
+
+/**
+ * Created by joshuakeough on 10/13/16.
+ */
+@Entity
+@Table(name = "reviews")
+public class Review {
+    @Id
+    @GeneratedValue
+    private int id;
+
+    @Column(nullable = false)
+    private String author;
+
+    @Column(nullable = false)
+    private String text;
+
+    @Column(nullable = false)
+    private int lecturerId;
```

Comment:
Glad to see you thought of adding the method findByLecturerId(). However, this did not work, did it?

If the review is to be tied to the Lecturer shouldn't you be using the @ManyToOne annotation with a Lecture type field? Although the UI does call an individual Lecturer's reviews this assignment does not require that route so the general findAll() works okay.